hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataNode.java

```diff
@@ -2227,14 +2227,14 @@ public void shutdown() {
     // wait reconfiguration thread, if any, to exit
     shutdownReconfigurationTask();
 
-    LOG.info("Waiting up to 30 seconds for transfer threads to complete");
+    LOG.info("Waiting up to 15 seconds for transfer threads to complete");
```

Comment:
`executorService.awaitTermination(timeout, unit)` will be possibly invoked 2 times in `HadoopExecutors#shutdown`. The time value in log looks ok.